be/src/exec/pipeline/pipeline_driver_dispatcher.cpp

```diff
@@ -45,7 +49,9 @@ void GlobalDriverDispatcher::run() {
 
         size_t queue_index;
         auto driver = this->_driver_queue->take(&queue_index);
-        DCHECK(driver != nullptr);
+        if (driver == nullptr) {
```

Comment:
_driver_queue always return a non-null driver
"DCHECK(driver != nullptr);"  is ok